ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/eos_cli_config_gen.schema.yml

```
@@ -93,6 +93,121 @@ keys:
           description: 'Action as string
 
             Example: "permit GSHUT 65123:123"'
+  maintenance:
+    type: dict
+    display_name: Profiles and units
+    keys:
+      default_interface_profile:
+        type: str
+        description: 'Name of default Interface Profile
+
+          '
+      default_bgp_profile:
+        type: str
+        description: 'Name of default BGP Profile
+
+          '
+      default_unit_profile:
+        type: str
+        description: 'Name of default Unit Profile
+
+          '
+      interface_profiles:
+        type: list
+        primary_key: name
+        convert_types:
+        - dict
+        items:
+          type: dict
+          keys:
+            name:
+              type: str
+              required: true
+            rate_monitoring:
+              type: dict
+              keys:
+                load_interval:
+                  type: int
+                  description: 'Load Interval in Seconds
+
+                    '
+                threshold:
+                  type: int
+            shutdown:
+              type: dict
+              keys:
+                max_delay:
+                  type: int
+      bgp_profiles:
```

Because of acronym, we still want the display name here
```suggestion
      bgp_profiles:
        display_name: BGP Profiles
```